parser/src/main/scala/org/globalnames/parser/Preprocessor.scala

```diff
@@ -63,9 +63,8 @@ object Preprocessor {
 
   private final val virusPatterns =
     """\sICTV\s*$""".r :: """[A-Z]?[a-z]+virus\b""".r ::
-      """(?ix)\b(virus|viruses|particle|particles|
-               phage|phages|viroid|viroids|virophage|
-               prion|prions|NPV)\b""".r ::
+      """(?ix)\b(virus(es)?|particle(s)?|(bacterio|viro)?phage(s)?|
```

Comment:
Why do you need to put single character `s` to parentheses? Regex engine creates a group for it, i.e. consumes more CPU cycles and memory.